app/views/layouts/_top_menu.html.erb

```diff
@@ -41,6 +41,10 @@
             <%= nav_link "Inventory", item_categories_path if can? :read, Item %>
           </li>
 
+          <li>
```

Comment:
Could you please move this under the inventory menu item ? I don't think it warrants a top level spot. 
See Inactive, under People for an example.
Keep the permissions check, that's good.

Reply:
👍